### ISSUE TITLE
Unbalance FGW full rank marginal normalization issue

### ISSUE THREAD

**Issue by giovp** [theislab/moscot#691]
xlink https://github.com/theislab/moscot/pull/686

**Comment by selmanozleyen** [comment]
Ok just to update this issue waits for https://github.com/ott-jax/ott/issues/519 to be closed and then we'd need to **unskip** the test related in
```python
class TestSpatioTemporalProblem:
...
@pytest.mark.skip(reason="unbalanced does not work yet: https://github.com/ott-jax/ott/issues/519")
    def test_solve_unbalanced(self, adata_spatio_temporal: AnnData):
```

**Comment by MUCDK** [comment]
Hi @selmanozleyen ,

What's the current state on this?

**Comment by selmanozleyen** [comment]
@MUCDK  its still the same, it is an error from the ott-jax side and the issue is linked above

**Comment by HarryZhang1224** [comment]
Hi @MUCDK @selmanozleyen,

Is unbalanced FGW still not supported?

Thanks

**Comment by MUCDK** [comment]
Hi @HarryZhang1224 ,

Unfortunately, this is an issue inherent to entropic FGW, see https://github.com/ott-jax/ott/issues/519.
Hence, you can run it in an unbalanced way, but make sure that the marginals sum up to 1.0.

**Comment by HarryZhang1224** [comment]
> Hi [@HarryZhang1224](https://github.com/HarryZhang1224) ,
> 
> Unfortunately, this is an issue inherent to entropic FGW, see [ott-jax/ott#519](https://github.com/ott-jax/ott/issues/519). Hence, you can run it in an unbalanced way, but make sure that the marginals sum up to 1.0.

@MUCDK Thanks for the quick reply! Specifying the marginals a and b to sum to 1 worked. Is there a numerical instability issue with unbalanced + entropic FGW? For large values of fused penalty, the returned coupling has all nan, and sometimes it also seems to interact with epsilon.

**Comment by MUCDK** [comment]
Yes, unfortunately, GW and FGW are sometimes hard to optimise. Normally, increasing epsilon would help, as this turns the problem near-convex, see

Gabriel Rioux, Ziv Goldfeld, and Kengo Kato. Entropic gromov-wasserstein distances: Stability,
algorithms, and distributional limits. arXiv preprint arXiv:2306.00182, 2023. 4
